### PR TITLE
i18n(fr): Improvements to French language support

### PR DESCRIPTION
**Pull request by PTrottier**
Thanks for the suggestions @CrimsonHemlock in #46, I agree, cheers! Added them as a separate PR to avoid rewriting to the existing PR.

Note, I changed all occurrences of `clé` for `touche` to help with consistency.

### CODE REVIEW

File: i18n/fr/system76_keyboard_configurator_backend.ftl [hunk 1]
```diff
@@ -1,7 +1,7 @@
-mode-disabled = Déactiver
+mode-disabled = Désactiver
 mode-solid-color = Couleur solide par couche
-mode-per-key = Solide par clé
-mode-active-keys = Clés active seulement
+mode-per-key = Solide par touche
+mode-active-keys = Touches active seulement
```
Comment (CrimsonHemlock) [inline i18n/fr/system76_keyboard_configurator_backend.ftl:4]: Just a small grammatical correction: it should be `Touches actives seulement`

**Comment by CrimsonHemlock** [comment]
> Thanks for the suggestions @CrimsonHemlock in #46, I agree, cheers! Added them as a separate PR to avoid rewriting to the existing PR.
> 
> Note, I changed all occurrences of `clé` for `touche` to help with consistency.

Thanks! Happy to help!

**Comment by PTrottier** [comment]
Thanks again, @CrimsonHemlock!! I also corrected another spelling mistake with [`Number of runs`](https://github.com/pop-os/keyboard-configurator/pull/49/commits/cd859d4afa07a0a2d1b5951281699ab760c198c2#diff-59eddc504519cb7c866997b3350563d0cb0ffa50feef01058cd4010ab32d8376R73)

**Comment by ids1024** [comment]
Rebased and merged as c94cee073285772889e5d8a162f80292057f01ab. Thanks again!

**Pull request closed without merging**